devcycle_python_sdk/api/local_bucketing.py

```diff
@@ -236,6 +240,19 @@ def _new_assembly_script_byte_array(self, param: bytes) -> int:
             for i, b in enumerate(param):
                 data[buffer_pointer + i] = b
 
+            # logger.warning(
+            #     f"header should contain: {buffer_pointer:04x} {buffer_pointer:04x} {data_length:04x}"
+            # )
+            # logger.warning(
+            #     f"buffer should contain: {' '.join([f'{data[i]:02x}' for i in range(buffer_pointer, buffer_pointer + data_length)])}"
+            # )
+            # logger.warning(
+            #     f"header: {' '.join([f'{data[i]:02x}' for i in range(header_pointer, header_pointer + 12)])}"
+            # )
+            # logger.warning(
+            #     f"buffer: {' '.join([f'{data[i]:02x}' for i in range(buffer_pointer, buffer_pointer + data_length)])}"
+            # )
+
```

Comment:
I think we can just take these logs out instead of keeping them around in a comment.

Reply:
lol, forgot but these.  commented them out while testing